modules/apps/commerce/commerce-product-test/src/testFunctional/paths/CommerceEntry.path

```diff
@@ -493,6 +493,17 @@
 	<td>//h4[contains(.,'Images')]//..//button[contains(@class,'btn-primary')]</td>
 	<td></td>
 </tr>
+<!--ATTACHMENT-->
+<tr>
+	<td>ATTACHMENT_PLUS_BUTTON</td>
+	<td>//h4[contains(.,'Attachments')]//..//button[contains(@class,'btn-primary')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>ATTACHMENT_MODAL_IFRAME</td>
+	<td>//form[@id='_com_liferay_commerce_product_definitions_web_internal_portlet_CPDefinitionsPortlet_fm']</td>
+	<td></td>
+</tr>
```

Comment:
This macro looks like that was copy/pasted from this other PR … so, please, cherrypick the code that the macro was created, to avoid the copy/paste way, as it can cause a rebase problems